_universum/modules/vcs/perforce_vcs.py

```diff
@@ -263,7 +263,9 @@ def __init__(self, *args, **kwargs):
         super(PerforceMainVcs, self).__init__(*args, **kwargs)
 
         if not getattr(self.settings, "client", None):
-            raise IncorrectParameterError("P4CLIENT (-p4c) is not specified. It is required for creating workspace.")
+            raise IncorrectParameterError("P4CLIENT (-p4c) is not specified.\n\n"
```

Comment:
I think it'd be better to update this message in accordance to other messages in this commit. Like, "perforce workspace (client) is not specified, it is used for bla-bla, please use command line option bla-bla or environment variable bla-bla". Or alternatively start every other error message with env. var. / option required, so that all of them look alike.